src/Compilers/VisualBasic/Portable/Symbols/Retargeting/RetargetingMethodSymbol.vb

```diff
@@ -157,13 +157,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Retargeting
 
         Public Overrides ReadOnly Property ReturnTypeCustomModifiers As ImmutableArray(Of CustomModifier)
             Get
-                Return RetargetingTranslator.RetargetModifiers(_underlyingMethod.ReturnTypeCustomModifiers, _lazyCustomModifiers)
+                Return RetargetingTranslator.RetargetModifiers(_underlyingMethod.ReturnTypeCustomModifiers, _underlyingMethod.RefCustomModifiers, _lazyCustomModifiers).TypeCustomModifiers
             End Get
         End Property
 
-        Friend Overrides ReadOnly Property CountOfCustomModifiersPrecedingByRef As UShort
+        Public Overrides ReadOnly Property RefCustomModifiers As ImmutableArray(Of CustomModifier)
             Get
-                Return _underlyingMethod.CountOfCustomModifiersPrecedingByRef
+                Return RetargetingTranslator.RetargetModifiers(_underlyingMethod.ReturnTypeCustomModifiers, _underlyingMethod.RefCustomModifiers, _lazyCustomModifiers).RefCustomModifiers
             End Get
```

Comment:
Consider adding a private `CustomModifierTuple` property so `_lazyCustomModifiers` is calculated in one place. Same comment for other retargeting symbols.